src/js/store/orgs/reducer.ts

```diff
@@ -27,6 +27,7 @@ export interface Org {
   has_unsaved_changes: boolean;
   currently_refreshing_changes: boolean;
   currently_capturing_changes: boolean;
+  currently_refreshing_org: boolean;
   delete_queued_at: string | null;
 }
 
```

Comment:
We need to add some handling to this reducer (below), so that it updates accordingly when the events are dispatched. With some minor tweaking to adjust to the payload and store structure etc, I think it's basically like this: https://github.com/oddbird/MetaShare/pull/162/files#diff-c78c71991afefd60e912491219e8a658R97-R116